packages/PlanLimitsUI/src/pages/Limits/compileLimitsTable.tsx

```diff
@@ -0,0 +1,106 @@
+import formatWaterQuantity from './formatWaterQuantity';
+import type { GroundwaterZoneBoundariesProperties } from '../../api';
+import type { FeatureCollection, Geometry } from 'geojson';
+
+const BLANK_CELL_CHAR = '-';
+const HEADERS = ['Type', 'Bore screen depth', 'Category'];
+const LIMIT_HEADERS = ['Allocation limit'];
+const LIMIT_HEADERS_WITH_CATCHMENT = [
+  'Sub-unit Allocation Limit',
+  'Catchment Management Unit Allocation Limit',
+];
+
+export default function compileLimitsTable(
+  surfaceWaterMgmtUnitId: number,
+  surfaceWaterMgmtUnitLimit: string | null | undefined,
+  surfaceWaterMgmtSubUnitLimit: string | null | undefined,
+  activeZonesIds: Array<number>,
+  groundWaterZoneGeoJson: FeatureCollection<
+    Geometry,
+    GroundwaterZoneBoundariesProperties
+  >
+) {
+  let showFootnote = false;
+  const rows = [
+    [
+      'Surface water',
+      BLANK_CELL_CHAR,
+      BLANK_CELL_CHAR,
+      <>{surfaceWaterMgmtSubUnitLimit || surfaceWaterMgmtUnitLimit}</>,
+      surfaceWaterMgmtUnitLimit,
+    ],
+  ];
+
+  const activeFeatures = groundWaterZoneGeoJson.features
+    .filter((item) => activeZonesIds.includes(Number(item.id as string)))
+    .map((feature) => {
+      return {
+        gwAllocationAmount: feature.properties.groundwater_allocation_amount,
+        gwAllocationUnit: feature.properties.groundwater_allocation_amount_unit,
+        depth: feature.properties.depth,
+        category: feature.properties.category,
+        swAllocationUnit:
+          feature.properties.surface_water_allocation_amount_unit,
+        swAllocationAmount: feature.properties.surface_water_allocation_amount,
+      };
+    });
+
+  activeFeatures
+    .filter((feature) => feature.category === 'Category A')
+    .forEach((feature) => {
+      const limit = formatWaterQuantity(
+        feature.swAllocationAmount,
+        feature.swAllocationUnit
+      );
+      rows.push([
+        'Groundwater',
+        feature.depth,
+        'A',
+        limit,
+        surfaceWaterMgmtUnitLimit,
+      ]);
+    });
+
+  activeFeatures
+    .filter((feature) => feature.category === 'Category B')
+    .forEach((feature) => {
+      showFootnote = true;
+      rows.push([
+        'Groundwater',
+        feature.depth,
+        'B',
+        <>
+          See Table 4.1 of PNRP
+          <sup>
+            <a href="#PNRP">1</a>
+          </sup>
+        </>,
+        '-',
+      ]);
+    });
+
+  activeFeatures
+    .filter((feature) => feature.category === 'Category C')
+    .forEach((feature) => {
+      const limit = formatWaterQuantity(
+        feature.gwAllocationAmount,
+        feature.gwAllocationUnit
+      );
+      rows.push(['Groundwater', feature.depth, 'C', limit, '-']);
+    });
+
+  // TODO: What is the best way to capture this requirement to show Catchement Level
+  // limits for particular units?
```

Comment:
I was thinking, that its just if there is both a limits we show both columns, which will only happen in those two zones anyways.

Reply:
I'm not quite sure that would work the same, since there are some map areas within these management units that are not within sub-management units. So both limits would not be shown when hovering over these, where currently both limits are shown anywhere within the whole management unit. I could be mis-understanding the data or your suggestion though, so let's talk it through.